### PR TITLE
doc/cephadm: edit troubleshooting.rst (1 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Edit doc/cephadm/troubleshooting.rst. This commit and the PR of which it is a part was raised in response to
https://github.com/ceph/ceph/pull/53976. The limits of reStructuredText are particularly visible here in every instance of a BASH for-loop and in every instance of a command stretched over multiple lines.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/cephadm/troubleshooting.rst [hunk 1]
```diff
@@ -1,60 +1,56 @@
...
 
-You may wish to investigate why a cephadm command failed
-or why a certain service no longer runs properly.
+This section explains how to investigate why a cephadm command failed or why a
+certain service no longer runs properly.
 
-Cephadm deploys daemons within containers. This means that
-troubleshooting those containerized daemons will require
-a different process than traditional package-install daemons.
+Cephadm deploys daemons within containers. Troubleshooting those containerized
```
Comment (anthonyeleven) [inline doc/cephadm/troubleshooting.rst:7]: s/those//

File: doc/cephadm/troubleshooting.rst [hunk 2]
```diff
@@ -1,60 +1,56 @@
...
 .. prompt:: bash #
 
   ceph orch set backend ''
   ceph mgr module disable cephadm
 
-These commands disable all of the ``ceph orch ...`` CLI commands.
-All previously deployed daemon containers continue to exist and
-will start as they did before you ran these commands.
+These commands disable all of the ``ceph orch ...`` CLI commands. All
+previously deployed daemon containers continue to exist and will start just as
```
Comment (anthonyeleven) [inline doc/cephadm/troubleshooting.rst:36]: s/exist/run/

File: doc/cephadm/troubleshooting.rst [hunk 3]
```diff
@@ -139,39 +144,41 @@ To fetch all log files of all daemons on a given host, run::
...
 
+   systemctl status "ceph-$(cephadm shell ceph fsid)@<service name>.service";
 
-To fetch all state of all daemons of a given host, run::
 
-    fsid="$(cephadm shell ceph fsid)"
-    for name in $(cephadm ls | jq -r '.[].name') ; do
-      systemctl status "ceph-$fsid@$name.service" > $name;
-    done
+To fetch all state of all daemons of a given host, run the following shell
```
Comment (anthonyeleven) [inline doc/cephadm/troubleshooting.rst:154]: s/all/the/

**Review by anthonyeleven — APPROVED** [review]

File: doc/cephadm/troubleshooting.rst (hunk 1 above)
Comment (zdover23) [reply doc/cephadm/troubleshooting.rst:7]: Accepted.

File: doc/cephadm/troubleshooting.rst (hunk 2 above)
Comment (zdover23) [reply doc/cephadm/troubleshooting.rst:36]: Accepted.

File: doc/cephadm/troubleshooting.rst (hunk 3 above)
Comment (zdover23) [reply doc/cephadm/troubleshooting.rst:154]: Oh, good God. How did I miss that?

Accepted.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/54283 - Reef backport
https://github.com/ceph/ceph/pull/54284 - Quincy backport

File: doc/cephadm/troubleshooting.rst (hunk 3 above)
Comment (anthonyeleven) [reply doc/cephadm/troubleshooting.rst]: Arguably subtle.  I had to read the context carefully to decide if it was in fact correct. There could be command variants that show all state of some daemons or some state of all daemons etc.

Comment (zdover23) [reply doc/cephadm/troubleshooting.rst]: I think it was just a matter of reading over 300 lines of text for a couple of hours, until the text became so stale that I needed a second pair of eyes.

Comment (anthonyeleven) [reply doc/cephadm/troubleshooting.rst]: Fortunately I have two of those.

**Pull request merged**